src/handlers.js

```diff
@@ -137,14 +147,30 @@ async function statusHandler(message) {
     state = 'failure';
   }
 
-  await build.modify((b) => {
-    b.state = state;
-    b.updated = new Date();
-  });
+  try {
+    debug('Trying to modify the build...');
+    await build.modify((b) => {
+      b.state = state;
+      b.updated = new Date();
+    });
+  } catch (e) {
+    debug('Failed to modify the build!');
+    throw e;
+  }
+
+  // Authenticating as installation.
+  try {
+    debug('Authenticating as installation in status handler...');
+    var instGithub = await this.context.github.getInstallationGithub(build.installationId);
```

Comment:
Similarly here.  I think you could leave a `debug()` in here, but add the `installationId` to the message.